.env.development

```diff
@@ -0,0 +1 @@
+GOVERNANCE_URL='https://xd.adobe.com/view/31a3d2a5-9f07-4e31-a612-20059ff929a5-64f0/screen/d8a45cb1-7433-40b3-b6ce-4237bfcd0678/?fullscreen'
```

Comment:
Should be carried by the dataspace, imho. See [api/fakeData/dataspaces/%5Bid%5D.ts#L4](https://github.com/okp4/portal-web/blob/main/src/pages/api/fakeData/dataspaces/%5Bid%5D.ts#L4)

Reply:
You are absolutely right. I added a `governance` property to `Dataspace` type which is a `string `for now, in order to match the v0 attempts for the expected link.

Reply:
As agreed, to allow a maximum of configuration in the current context, I propose to "externalize" all the json that describe the dataspace, services and dataset. Coordinate with  @SonnevilleDamien. A PR is in progress...

Reply:
We can merge this PR as it is.